main.py

```diff
@@ -0,0 +1,29 @@
+from github import Auth
+from github import Github
+import os 
+class GithubData:
+    def __init__(self, repo, access_token) -> None:
+        auth = Auth.Token(access_token)
+        self.github = Github(auth=auth)
+        self.repo = self.github.get_repo(repo)
+
+    def get_pr_diff(self, pr_number):
```

Comment:
Using mutable default arguments in functions can lead to unexpected behaviors. Ensure that mutable objects like lists or dictionaries are not used as default arguments. Although this case doesn't directly use mutable defaults, it's good practice to be cautious about it.